src/common/com/intellij/plugins/haxe/lang/util/HaxePsiUtil.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
How about we rename this file HaxeExpressionUtil?  We already have a number of PsiUtil classes and they are used for inspecting and manipulating the PSI.  All of these functions are about expressions anyway.